### PR TITLE
Add a webview module

### PR DESCRIPTION
**Pull request by shadeslayer**
Add a webview module so that calamares can load a webpage.
This is useful when configuring webservices on your device such as
ownCloud.

### CODE REVIEW

**Comment by shadeslayer** [comment]
Screenshot in case anyone is interested

image

**Comment by demmm** [comment]
Can this be connected to the Welcome page links too?  So no need then to launch an external browser to view "Known issues" for example?

**Comment by shadeslayer** [comment]
@demm one could make it so that it loads the release notes when you click next from the welcome page.

File: src/modules/webview/WebViewStep.cpp [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/* === This file is part of Calamares - <http://github.com/calamares> ===
+ *
+ *   Copyright 2014-2015, Teo Mrnjavac <teo@kde.org>
```
Comment (teo) [inline src/modules/webview/WebViewStep.cpp]: Please add your copyright line.

File: src/modules/webview/WebViewStep.cpp [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+WebViewStep::isNextEnabled() const
+{
+    return true;
+}
+
+
+bool
+WebViewStep::isBackEnabled() const
+{
+    return false;
```
Comment (teo) [inline src/modules/webview/WebViewStep.cpp]: This should probably be `true`.

File: src/modules/webview/WebViewStep.h [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/* === This file is part of Calamares - <http://github.com/calamares> ===
+ *
+ *   Copyright 2014-2015, Teo Mrnjavac <teo@kde.org>
```
Comment (teo) [inline src/modules/webview/WebViewStep.h]: Copyright line.

**Comment by shadeslayer** [comment]
@teo fixed all the issues

**Comment by kkofler** [comment]
Shouldn't this module with its QtWebKit dependency be optional (compile-time option)?

**Comment by teo** [comment]
@kkofler it is already possible to exclude certain modules from building with -DSKIP_MODULES

**Pull request merged**